admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

```diff
@@ -112,61 +112,71 @@
     <script type="text/javascript" src="${item}"></script>
 </sly>
 
-<sly data-sly-include="renderer.html"></sly>
+<sly data-sly-include="renderer.html" />
 <script>
+    var $perAdminView = {
+        state: {},
+        adminPage: {},
+        adminPageStaged: {},
+        admin: {},
+        pageView: {
+            view: 'desktop'
+        },
+        nodes: {}
+    }
+
     // switch between old and new implementation
     var newImpl = true
-    var $perAdminView = { state: {}, adminPage: {}, adminPageStaged: {}, admin: {}, pageView: { view: 'desktop'}, nodes: {} }
-
-    if(newImpl) {
+    if (newImpl) {
+        var loadComponent = $perAdminApp.loadComponent;
```

Comment:
not sure I like this one - a var in javascript is global and const are not supported in all older browsers

consider this script:
```
<script>
	if(true) {
		var test = 'am I surviving my block?';
	}
</script>

<script>
	console.log(test)
</script>
```

the output will be

```am I surviving my block?```

hence this change may have side effects

Reply:
Changed, please check.